### PR TITLE
.Net Core 2.1

### PR DESCRIPTION
**Pull request by kblohm**
Hi,
as .Net Core 2.1 was just released, i updated FAKE to use the new version.
I split the update in 2 commits:
1. Adds the new version to the DotNet-Module and uses the release version in the build instead of the RC.
2. Replaces netcore2.0 with 2.1. I am not sure if we want to do this already or maybe just do it for tests and not for fake-dotnet. If we do not want to update yet or not update  the tools, i can revert/adjust this one.

### CODE REVIEW

**Comment by matthid** [comment]
Yes the only interesting impact is dotnet-fake because it will no longer work when people have 2.0. Everything else probably doesn't matter that much and we can upgrade

**Comment by 0x53A** [comment]
What is the advantage to updating? Breaking 2.0 without any reason seems .... not good?

**Comment by kblohm** [comment]
Updating dotnet-fake has no real advandate as far as i can tell. I read that you have to rebuild global cli tools when the full version is released, so fake-cli might not work anymore now for some people. Its working for me though, so i am not sure if that is actually the case.

**Comment by matthid** [comment]
@0x53A Updating everything else besides "dotnet-fake" should be "safe" and gets all the performance and runtime improvements.

File: src/app/Fake.netcore/Fake.netcore.fsproj [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
-      <FromP2P>true</FromP2P>
-    </ProjectReference>
-  </ItemGroup>
-  <Import Project="..\..\..\.paket\Paket.Restore.targets" />
-</Project>
\ No newline at end of file
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" 
+    xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <TargetFramework>netcoreapp2.1</TargetFramework>
```
Comment (matthid) [inline src/app/Fake.netcore/Fake.netcore.fsproj]: any idea why every line changed? Line ending changed?

File: src/app/dotnet-fake/dotnet-fake.fsproj [hunk 2]
```diff
@@ -1,30 +1,31 @@
...
-      <FromP2P>true</FromP2P>
-    </ProjectReference>
-  </ItemGroup>
-  <Import Project="..\..\..\.paket\Paket.Restore.targets" />
-</Project>
\ No newline at end of file
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" 
+    xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <TargetFramework>netcoreapp2.1</TargetFramework>
```
Comment (matthid) [inline src/app/dotnet-fake/dotnet-fake.fsproj]: I think we should consider leaving netcoreapp2.0 (or both) here

File: src/app/Fake.netcore/Fake.netcore.fsproj (hunk 1 above)
Comment (kblohm) [reply src/app/Fake.netcore/Fake.netcore.fsproj]: Auto-Format on save in vscode. Probably uses the 4 spaces from the editorconfig.

File: src/app/dotnet-fake/dotnet-fake.fsproj (hunk 2 above)
Comment (kblohm) [reply src/app/dotnet-fake/dotnet-fake.fsproj]: I will change that later today when i am at home. Do you want both or just 2,0?

File: src/app/Fake.netcore/Fake.netcore.fsproj (hunk 1 above)
Comment (matthid) [reply src/app/Fake.netcore/Fake.netcore.fsproj]: Ah I didn't noticed, thanks. Can we change that for project files to two? Or just leave it as is. Now that I understand whats going on :)
But I think two is what the files use when auto generated (for example via `dotnet new`), correct?

File: src/app/dotnet-fake/dotnet-fake.fsproj (hunk 2 above)
Comment (matthid) [reply src/app/dotnet-fake/dotnet-fake.fsproj]: Thanks! I guess leaving 2.0 for now is the better option. We just wait until people start complaining.

**Comment by kblohm** [comment]
I also added *.fsproj to the editorconfig to fix the indentation issue.
If you want me to squash or something, ping :).

**Comment by parthopdas** [comment]
@matthid can you please push this out to nuget?

**Comment by matthid** [comment]
@kblohm Thanks for taking care of this!
@parthopdas Yes sure, once we made sure everything works on staging ;)

**Pull request merged**